roundabout-core/src/main/java/at/fhv/itm3/s2/roundabout/entity/Route.java

```diff
@@ -63,7 +63,7 @@ public boolean isEmpty() {
 
     public int getIndexOfSection(Street streetSection) {
         if (!route.contains(streetSection)) {
-            throw new IllegalArgumentException("Section must be part of the route");
+            throw new IllegalArgumentException("Track must be part of the route");
```

Comment:
why track and not section?

Reply:
because in xml its called track and this is an xml structure error